apps/condo/domains/acquiring/schema/RecurrentPayment.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Generated by `createschema acquiring.RecurrentPayment 'status:Text;tryCount:Integer;state:Json;billingReceipts:Json'`
+ */
+
+const { Relationship, Integer, Select } = require('@keystonejs/fields')
+
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/acquiring/access/RecurrentPayment')
+const {
+    RECURRENT_PAYMENT_STATUSES,
+    RECURRENT_PAYMENT_INIT_STATUS,
+} = require('@condo/domains/acquiring/constants/recurrentPayment')
+
+
+const RecurrentPayment = new GQLListSchema('RecurrentPayment', {
+    schemaDoc: 'Hold a snapshot of monthly payment data. Used by background recurrent payment process.',
+    fields: {
+        status: {
+            schemaDoc: 'Payment proceeding status',
+            type: Select,
+            dataType: 'enum',
+            options: RECURRENT_PAYMENT_STATUSES,
+            defaultValue: RECURRENT_PAYMENT_INIT_STATUS,
+        },
+
+        tryCount: {
+            schemaDoc: 'Count of proceeding tries.',
+            type: Integer,
+            isRequired: true,
+        },
+
+        state: {
+            schemaDoc: 'Hold current state of payment proceeding.',
+            type: Json,
+            isRequired: true,
+        },
+
+        billingReceipts: {
+            schemaDoc: 'Array of billing receipt ids to pay',
+            type: Json,
```

Comment:
Should all this JSON fields (or at least some of them) be typed?

Reply:
State/settings - are too complex for typed types
billingReceipts - is array of billing ids
I believe we have to use json type for all of them

Reply:
I think billing receipts should be typed. We did not type `state` and `settings` in `acquiringContext` or `billingContext` because of 3rd party integraions.

Reply:
^^